src/lib/isIMEI.js

```diff
@@ -1,15 +1,29 @@
 import assertString from './util/assertString';
 
 
-let imeiRegex = /^[0-9]{15}$/;
+let imeiRegexWithoutHypens = /^[0-9]{15}$/;
+let imeiRegexWithHypens = /^\d{2}-\d{6}-\d{6}-\d{1}$/;
 
-export default function isIMEI(str) {
+
+export default function isIMEI(str, options) {
   assertString(str);
+  options = options || {};
+
+  // default regex for checking imei is the one without hyphens
+
+  let imeiRegex = imeiRegexWithoutHypens;
+
+  if (options.allow_hyphens) {
```

Comment:
I have not tested this but does it mean that if `allow_hyphens` is `true` then it won't accept IMEI without hyphens?

Reply:
basically the same imei can be represented as   `###############` or `##-######-######-#`.
so if allow hyphens is set to true, the same validator will also the imei of second format @profnandaa

Reply:
For clarity , you can look at the test cases @profnandaa

Reply:
@profnandaa  any updates on this one?